CHANGELOG.md

```diff
@@ -39,6 +39,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v6.4.0](https://github.com/osmosis-labs/osmosis/releases/tag/v6.4.0)
+
+## Minor improvements & Bug Fixes
+
+-[#907](https://github.com/osmosis-labs/osmosis/pull/907) Upgrade IAVL and SDK with RAM improvements and bug fixes for v6.4.0
+
+### SDK fork updates
+
+- [sdk-#114](https://github.com/osmosis-labs/cosmos-sdk/pull/114) [commit](https://github.com/osmosis-labs/cosmos-sdk/pull/114/commits/7ec05684856bf87b868f67d05459d64931ac599d) upgrading iavl with ram optimizations during migration and extra logs
```

Comment:
Ah sorry I didn't click through to the PR before, didn't realize its the bug fix. So then we just need to update this log message.
```suggestion
- [sdk-#114](https://github.com/osmosis-labs/cosmos-sdk/pull/114) upgrading iavl with ram optimizations during migration and extra logs
- [commit](https://github.com/osmosis-labs/cosmos-sdk/pull/114/commits/7ec05684856bf87b868f67d05459d64931ac599d) Fix the bug when a node crashes during commit, and runs into non-logical commit issues.
```

Reply:
Ah I see what you meant